src/test/java/uk/ac/ebi/eva/contigalias/controller/contigalias/ContigAliasHandlerTest.java

```diff
@@ -62,86 +59,6 @@ public class ContigAliasHandlerTest {
-
-        private final long TOTAL_CHROMOSOMES = 27;
-
-        @BeforeEach
-        public void setup() {
-            handler = new ContigAliasHandler(null, null, null, null, null);
-        }
-
-        @Test
-        void createScaffoldsPageRequestTestOnlyChromosomes() {
-            PageRequest request = PageRequest.of(1, 10);
-            List<Pageable>[] scaffoldsPageRequest = handler.createScaffoldsPageRequest(TOTAL_CHROMOSOMES, request);
-
-            assertNotNull(scaffoldsPageRequest);
-
-            List<Pageable> chrRequests = scaffoldsPageRequest[0];
-            assertNotNull(chrRequests);
-            assertEquals(1, chrRequests.size());
-
-            Pageable chrRequest = chrRequests.get(0);
-            assertEquals(1, chrRequest.getPageNumber());
-            assertEquals(10, chrRequest.getPageSize());
-
-            List<Pageable> scfRequests = scaffoldsPageRequest[1];
-            assertNotNull(scfRequests);
-            assertEquals(0, scfRequests.size());
-        }
-
-        @Test
-        void createScaffoldsPageRequestTestOnlyScaffolds() {
-            PageRequest request = PageRequest.of(3, 10);
-            List<Pageable>[] scaffoldsPageRequest = handler.createScaffoldsPageRequest(TOTAL_CHROMOSOMES, request);
-
-            assertNotNull(scaffoldsPageRequest);
-
-            List<Pageable> chrRequests = scaffoldsPageRequest[0];
-            assertNotNull(chrRequests);
-            assertEquals(0, chrRequests.size());
-
-            List<Pageable> scfRequests = scaffoldsPageRequest[1];
-            assertNotNull(scfRequests);
-            assertEquals(2, scfRequests.size());
-
-            Pageable scfRequest1 = scfRequests.get(0);
-            assertEquals(0, scfRequest1.getPageNumber());
-            assertEquals(7, scfRequest1.getPageSize());
-
-            Pageable scfRequest2 = scfRequests.get(1);
-            assertEquals(1, scfRequest2.getPageNumber());
-            assertEquals(3, scfRequest2.getPageSize());
-        }
-
-        @Test
-        void createScaffoldsPageRequestTestBothCombined() {
-            PageRequest request = PageRequest.of(2, 10);
-            List<Pageable>[] scaffoldsPageRequest = handler.createScaffoldsPageRequest(TOTAL_CHROMOSOMES, request);
-
-            assertNotNull(scaffoldsPageRequest);
-
-            List<Pageable> chrRequests = scaffoldsPageRequest[0];
-            assertNotNull(chrRequests);
-            assertEquals(1, chrRequests.size());
-
-            Pageable chrRequest = chrRequests.get(0);
-            assertEquals(2, chrRequest.getPageNumber());
-            assertEquals(7, chrRequest.getPageSize());
-
-            List<Pageable> scfRequests = scaffoldsPageRequest[1];
-            assertNotNull(scfRequests);
-            assertEquals(1, scfRequests.size());
-
-            Pageable scfRequest = scfRequests.get(0);
-            assertEquals(0, scfRequest.getPageNumber());
-            assertEquals(3, scfRequest.getPageSize());
-        }
-
-    }
-
```

Comment:
No more manual pagination since we are not separately returning results for Chromosome and Scaffolds in a custom paged fashion